scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -994,13 +1370,37 @@ String getTaskServerLogsByTag(String id, String taskTag)
     JobInfo getJobInfo(String jobId) throws UnknownJobException, NotConnectedException, PermissionException;
 
     /**
-     * Change the START_AT generic information at job level and reset the scheduledAt at task level
-     * @param jobId id of the job that needs to be updated
-     * @param startAt its value should be ISO 8601 compliant
+     * Change the START_AT generic information at job level and reset the
+     * scheduledAt at task level
+     * 
+     * @param jobId
+     *            id of the job that needs to be updated
+     * @param startAt
+     *            its value should be ISO 8601 compliant
      * @throws NotConnectedException
      * @throws UnknownJobException
      * @throws PermissionException
      */
     boolean changeStartAt(JobId jobId, String startAt)
             throws NotConnectedException, UnknownJobException, PermissionException;
+
+    /**
+     * retrieve a job content with the given job id, replace the general
+     * information with the given one, resubmit the job
+     * 
+     * @param jobId
+     *            job id used to retrieve the job content
+     * @param generalInfo
+     *            general information to replace or add to the job
+     * @return job id of the new created job
+     * @throws NotConnectedException
+     * @throws UnknownJobException
+     * @throws PermissionException
+     * @throws SubmissionClosedException
+     * @throws JobCreationException
+     */
+    JobId copyJobAndResubmitWithGeneralInfo(JobId jobId, Map<String, String> generalInfo)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.ow2.proactive.scheduler.common.exception.NotConnectedException, org.ow2.proactive.scheduler.common.exception.UnknownJobException, org.ow2.proactive.scheduler.common.exception.PermissionException, org.ow2.proactive.scheduler.common.exception.SubmissionClosedException, org.ow2.proactive.scheduler.common.exception.JobCreationException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)